src/ontology/uberon-edit.obo

```diff
@@ -224854,6 +224854,23 @@ relationship: composed_primarily_of CL:0000057 ! fibroblast
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-03T14:38:17Z xsd:dateTime
 
+[Term]
+id: UBERON:8480035
```

Comment:
I think we need to also take a look at [UBERON:0005805](http://purl.obolibrary.org/obo/UBERON_0005805) (dorsal aorta), [UBERON:0005294](http://purl.obolibrary.org/obo/UBERON_0005294) (gonadal ridge), and [UBERON:0000080](http://purl.obolibrary.org/obo/UBERON_0000080) (mesonephros) - all of which already have develops from axioms. These should all be coherent with this, and develops_from axioms need to be added where appropriate to link this term up correctly